Squest/utils/squest_views.py

```diff
@@ -1,14 +1,217 @@
+import traceback
+
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import PermissionDenied
+from django.db.models import ProtectedError
+from django.urls import reverse_lazy, NoReverseMatch, reverse
+from django.utils.safestring import mark_safe
+from django.views import View
+from django.views.generic import CreateView, UpdateView, DeleteView, DetailView, FormView
+from django.views.generic.detail import SingleObjectMixin
 from django_filters.views import FilterView
 from django_tables2 import SingleTableMixin
 
+from Squest.utils.squest_rbac import SquestPermissionRequiredMixin
+
+class SquestPermissionDenied(PermissionDenied):
+    def __init__(self, permission, *args, **kwargs):
+        self.permission = permission
+        super().__init__(mark_safe(f"Permission <b>{permission}</b> required"))
+
+
+class SquestView(View):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.django_content_type = ContentType.objects.get_for_model(self.model)
+
+    def get_generic_url_kwargs(self):
+        return {}
```

Comment:
bug with http://192.168.58.129:8000/ui/resource_tracker/attributes/1/delete/
The first breadcrum doesn't return the correct list URL

Reply:
done